corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -108,4 +108,8 @@
   }
 }
 
+[ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
+  display: none !important;
```

Comment:
This prevents flicker in the time between the initial page load and the first angular render.
https://docs.angularjs.org/api/ng/directive/ngCloak